astropy/table/tests/test_masked.py

```diff
@@ -195,10 +196,15 @@ def test_incomplete_mask_spec(self):
             MaskedColumn(name="b", length=4, mask=mask_list)
 
 
+STR_DTYPES = ["?", "b", "i2", "f4", "c8", "S", "U", "O"]
```

Comment:
`STR_DTYPES` is a confusing name since I read that as dtypes for string-like types. Maybe `DTYPES_TEST_INIT`. The `type_str` parameter in the test is a bit misleading now as well since some of the types are being specified as a class type not a str, but that can probably stay. Sigh.

Reply:
easy enough to also rename `type_str` to `dtype`. Thanks for pointing this out !